llvm/lib/Target/RISCV/RISCVInstrFormats.td

```diff
@@ -111,6 +112,7 @@ def OPC_BRANCH    : RISCVOpcode<0b1100011>;
 def OPC_JALR      : RISCVOpcode<0b1100111>;
 def OPC_JAL       : RISCVOpcode<0b1101111>;
 def OPC_SYSTEM    : RISCVOpcode<0b1110011>;
+def OPC_COREV_HWLP: RISCVOpcode<0b1111011>;
```

Comment:
I think this should be whichever `OPC_CUSTOMx` this actually is in the background, since other custom extensions will need this

Reply:
So should I rename this to `OPC_CUSTOM_COREV_HWLP`?